### PR TITLE
Update migration.md

### PR DESCRIPTION
**Pull request by mauroagr**
Add information about update using module 1.0.1

### CODE REVIEW

File: thirtybees/installation/migration.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: thirty bees Migration Module
+title: Thirty Bees Migration Module
```
Comment (Traumflug) [inline thirtybees/installation/migration.md]: Nope, 'thirty bees' is always all lowercase :-)

**Comment by Dh42** [comment]
Hey, thirty bees is always spelled lowercase, https://thirtybees.com/about/brand-guidelines/

File: thirtybees/installation/migration.md [hunk 2]
```diff
@@ -8,6 +8,20 @@ permalink: /installation/migration/
 ---
 
 # Thirty Bees - Migration from Prestashop 1.6 to Thirty Bees
+## Using Módule version 1.0.1 
+The migration is more easy, you can migrate direct from PS 1.6 to TB 1.0.6 direct
```
Comment (Traumflug) [inline thirtybees/installation/migration.md:12]: This `o` in 'Module' doesn't look like an English character.

**Comment by Traumflug** [comment]
Thank you very much for the contribution! I've added a pair of comments.

File: thirtybees/installation/migration.md (hunk 2 above)
Comment (mauroagr) [reply thirtybees/installation/migration.md:12]: ok...in Brazil is 'Módulo' with accent...sorry.

File: thirtybees/installation/migration.md (hunk 1 above)
Comment (mauroagr) [reply thirtybees/installation/migration.md]: thanks

**Review by mauroagr — COMMENTED** [review]
Updated

**Pull request closed without merging**